packages/hackmcx-api/src/routers/v1/posts/captions.router.js

```diff
@@ -1,7 +1,8 @@
 import express from "express";
-import {getCaptions, postCaptions} from "../../../controllers/captions.controller.js";
+import {getCaptions, postCaptions, rateCaption} from "../../../controllers/captions.controller.js";
 
 export const router = new express.Router({mergeParams: true});
 
 router.get('/', getCaptions)
-router.post('/', postCaptions)
\ No newline at end of file
+router.post('/', postCaptions)
+router.post("/:captionId/_rate", rateCaption)
```

Comment:
issue title has it being _rating

Reply:
I think it should be named `_rate`, I will go update that ticket.

Reply:
https://github.com/COSC481W-2022Winter/capstone-hackmcx/issues/59#issuecomment-1065990141